lib/carrierwave/storage/fog.rb

```diff
@@ -450,7 +450,10 @@ def filename(options = {})
         # @return [CarrierWave::Storage::Fog::File] the location where the file will be stored.
         #
         def copy_to(new_path)
-          connection.copy_object(@uploader.fog_directory, file.key, @uploader.fog_directory, new_path, acl_header)
+          options = {}
+          options.merge!(acl_header) if acl_header.present?
+          options['Content-Type'] ||= content_type if content_type
```

Comment:
I also saw that we're missing `@uploader.fog_attributes`. I think we should break this into a separate method? (https://github.com/carrierwaveuploader/carrierwave/pull/2504)

Reply:
Thank you, merged